geoportailv3/static/js/catalog/layertreenode.html

```diff
@@ -1,14 +1,14 @@
 <a data-toggle="collapse" data-parent="#catalog-{{::layertreenodeCtrl.parentUid}}"
    href="#catalog-{{::layertreenodeCtrl.uid}}"
    ng-if="::layertreenodeCtrl.node.children && layertreenodeCtrl.depth < 3">
-  {{::layertreenodeCtrl.node.name}}
+  {{::layertreenodeCtrl.node.name | translate}}
```

Comment:
And while we're at that we should remove the use of the `translate` filter in the index.html page: https://github.com/Geoportail-Luxembourg/geoportailv3/blob/master/geoportailv3/templates/index.html#L57.